demo/src/docs/Home.js

```diff
@@ -34,7 +29,7 @@ export default withStyles((theme) => ({
   container: {
     width: "100%",
     padding: "12px",
-    maxWidth: "800px",
+    maxWidth: "680px",
     margin: "45px auto",
```

Comment:
This is another one of the containers that are centered, where some are left aligned.